tests/util/test_builtin_operators.py

```diff
@@ -85,6 +85,23 @@ def test_regex_match(self):
         self.assertTrue(util.regex_match_func("/topic/delete/0", "/topic/delete/[0-9]+"))
         self.assertFalse(util.regex_match_func("/topic/edit/123s", "/topic/delete/[0-9]+"))
 
+    def test_glob_match(self):
+        self.assertTrue(util.glob_match_func("/foo", "/foo"))
```

Comment:
plz use comment to label which test is different from Go like:

```py
 self.assertTrue(util.glob_match_func("/foo", "/foo*")) # balabala
```

Reply:
Done !
I added all the tests done from Casbin Go on this globMatch function and marked the ones where the behavior differ.